ui/app/components/app/connected-accounts-permissions/index.scss

```diff
@@ -14,18 +14,18 @@
   }
 
   &__header {
+    @include H6;
+
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
     cursor: pointer;
-    font-size: 14px;
-    line-height: 20px;
     color: #24292e;
 
     button {
-      font-size: 16px;
-      line-height: 24px;
+      @include H5;
```

Comment:
There seems to be a specificity problem here; the "angle-down" symbol is now broken:

![broken-chevron](https://user-images.githubusercontent.com/2459287/89459952-e306b300-d73f-11ea-981c-d72773150a08.png)

The `font-weight` and `font-family` rules are overriding the ones from `fas` that are relied upon here

Reply:
The line height also differs slightly, but I don't think it matters in this case (to my eyes it looks marginally better, but it's barely noticeable)